packages/perl_sgmls.rb

```diff
@@ -1,36 +1,34 @@
-
 require 'package'
 
 class Perl_sgmls < Package
   description 'a set of Perl5 routines for processing the output from the onsgmls SGML parsers.'
   homepage 'http://search.cpan.org/dist/SGMLSpm/'
-  compatibility 'all'
   license 'GPL-1+ or Artistic'
-  version '1.1-1'
+  version '1.1-2'
```

Comment:
Can we put version above license?

Reply:
There has to be an awk or sed construction which should make this easy, right?

Reply:
This does a mediocre job in checking all files.
`for i in perl*.rb ; do gawk -i inplace '/^\slicense/{t=$0;next}1;/^\sversion/{print t}' $i ; rubocop -A $i;  done`